01-introduction.md

```diff
@@ -74,11 +75,19 @@ so that they get the positive feedback of being able to read whole stories as ea
 The basic idea is that once children find reading rewarding,
 they'll be motivated to go back and figure out the letters.
 
-Both approaches are consistent with what we know about how children learn,
-so we need to study them in action to determine which is better.
-The findings have been contradictory:
-while some studies find that phonics works best,
-others show that whole language yields better results.
+These approaches are not equally consistent with what we know about how children
+learn to read. While there is some evidence that proficient, adult readers
+identify whole word patterns, the scientific consensus among linguists,
+psychologists and neuroscientists is that in order for a child's brain to learn
+how to read, it is required to first make out the mapping of sounds to letters.
```

Comment:
On Wed, Jan 13, 2016 at 8:43 PM, W. Trevor King <notifications@github.com>
wrote:

> In 01-introduction.md
> <https://github.com/swcarpentry/instructor-training/pull/61#discussion_r49687246>
> :
>
> > @@ -74,11 +75,19 @@ so that they get the positive feedback of being able to read whole stories as ea
> >  The basic idea is that once children find reading rewarding,
> >  they'll be motivated to go back and figure out the letters.
> >
> > -Both approaches are consistent with what we know about how children learn,
> > -so we need to study them in action to determine which is better.
> > -The findings have been contradictory:
> > -while some studies find that phonics works best,
> > -others show that whole language yields better results.
> > +These approaches are not equally consistent with what we know about how children
> > +learn to read. While there is some evidence that proficient, adult readers
> > +identify whole word patterns, the scientific consensus among linguists,
> > +psychologists and neuroscientists is that in order for a child's brain to learn
> > +how to read, it is required to first make out the mapping of sounds to letters.
>
> Required seems too strong. For example the Bentin article we were
> discussing earlier <http://www.haskins.yale.edu/sr/SR111/SR111_13.pdf>,
> has:
>
> For example, using consonant addition and deletion tasks, Read, Zhang,
> Nie, and Ding (1986) found well-developed phonological awareness in Chinese
> subjects who learned to read a recently developed Chinese alphabetic system
> (Pinyin) but not among subjects who read only the logographic system
> (Kanji). The mean percentage of correct performance was 83% in the former
> group but only 21% in the later.
>
> So some kids are learning from logograms (they just aren't as good at
> consonant addition and deletion tasks).
>
>
I actually see that as evidence in favor of a strict phonics view. If you
learn an alphabetical system (=one that translates phonemes to symbols),
you will develop the phonological awareness required for skilled reading.
If you are learning a logographic system, you will not implicitly learn
about symbol <=> sound associations, necessary for skilled reading
[citation needed]. But I am still not sure.

BTW -- @wking -- I tried emailing you a pdf of the sciam article, but it
bounced off your email address. I'll slack it to the lessons channel.

And thanks for all the comments - very helpful!

> My take on the triangle model in the Byrd summary
> <https://books.google.com/books?id=IoTdAUdNkgIC&pg=PA302#v=onepage&q&f=false>
> was that good phonics helps you go from letters → sounds → meaning when you
> hit an unfamiliar word, which is useful if you already speak the language
> (and therefore have good sound ↔ meaning awareness. Maybe say something
> like:
>
> There is some evidence that proficient, adult readers identify whole word
> patterns. But humans are *really good* at associating sounds with
> meaning, and the scientific consensus among linguists,
> psychologists and neuroscientists is that the bridge phonics builds from
> letters to sounds makes learning to read easier [cite Byrd].
>
>



> —
> Reply to this email directly or view it on GitHub
> <https://github.com/swcarpentry/instructor-training/pull/61/files#r49687246>
> .
>